ewok_village/map.conf

```
@@ -0,0 +1,13 @@
+team.1.pos = (38,-10,39)
+r = 50
+initial_stuff = default:pick_steel, default:junglewood 50
```

```suggestion
initial_stuff = default:pick_steel,default:junglewood 50,default:torch 3
```
Don't need to put space. Also add torches